Yubico.YubiKey/src/Yubico/YubiKey/Scp/Commands/DeleteKeyCommand.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright 2023 Yubico AB
```

Comment:
wrong year

Reply:
I'm sure someone pinged Alessio's thread in Slack with legal on what to do here... FWIW - Microsoft's answer for Windows was to remove the year from all file headers. In the open source dotnet/runtime project, I don't see the year anywhere. I think a decent middle ground would be to take it out of all of the individual files and stick it in the main LICENSE or NOTICE file (or README). That's my 2 cents - also obligatory "I am not a lawyer" statement 😉